### ISSUE TITLE
Newly installed app's icon should be shown on right of the 'dot' until decided by user

### ISSUE THREAD

**Issue by ooopscc** [Mortennn/Dozer#67]

**Comment by half-baked1** [comment]
Yes! This is the reason I came here. When I record videos in QuickTime player, the button to end the recording ends up hidden, and I have to unhide all the icons the end of the recording.

**Comment by iDjay** [comment]
yes definitely needing that option :)

**Comment by Mortennn** [comment]
I need some help before I can implement this #69.
